lib/rubygems/specification.rb

```diff
@@ -1714,12 +1714,14 @@ def has_conflicts?
   end
 
   ##
-  # The date this gem was created.  Lazily defaults to the current UTC date.
+  # The date this gem was created.  Lazily defaults to the current UTC date or if
+  # SOURCE_DATE_EPOCH is set as an environment variable (unix timestamp) to
+  # support reproducible builds.
```

Comment:
@anthraxx Thoughts on changing this to something like the following?

```
# The date this gem was created.
#
# If SOURCE_DATE_EPOCH is set as an environment variable, use that to support
# reproducible builds; otherwise, default to the current UTC date.
#
# Details on SOURCE_DATE_EPOCH:
# https://reproducible-builds.org/specs/source-date-epoch/
```